css/templates/travel/components/_filter-bar.css

```diff
@@ -0,0 +1,16 @@
+.travel-filter-bar {
+  animation: fade-and-slideup 0.4s 0.05s ease-in-out both;
+}
+
+.travel-filter-bar-collapse {
+  height: 0;
+  overflow: hidden;
+}
+
+.travel-filter-bar .gray {
```

Comment:
would prefer not to use the bass color classes  let me know what you think. Also i would definitely stay away from this type of usage. just declare another class to override